website/content/docs/api-clients/api.mdx

```diff
@@ -79,3 +81,184 @@ The following method conventions are used within Boundary's API:
 ### DELETE
 
 `DELETE` is used for deleting a specific resource, and is only used against a particular resource path.
+
+## Rate limiting
+
+If your controllers try to process every API request, it can lead to situations when the controllers are overwhelmed with requests.
+The controllers may either run out of resources or overwhelm the database server and exhaust its resources.
+API rate limiting lets you configure limits on the rate of API requests to help manage your resources and prevent them from being overwhelmed.
+
+Boundary creates quotas to track the number of requests in a given time period.
+By default, Boundary tracks the number of requests by auth token, IP address, and the overall total.
+Boundary reserves a portion of its memory for tracking quotas to ensure that it does not consume too much memory if there is a sudden burst of requests.
+If Boundary is unable to store a quota, it limits the request with a 503 HTTP status code.
+You can configure the amount of memory that is reserved for tracking quotas using the `api_rate_limit_max_quotas` variable.
```

Comment:
I think this phrasing might be confusing. They are able to configure the number of quotas, which does relate to memory, but they cannot directly set an amount of memory here. I think the way this is currently phrased, operators might think they can set something like `api_rate_limit_max_quotas = 256MiB`.